alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -187,6 +191,21 @@ class AlertService(
         )
     }
 
+    fun composeMonitorErrorAlert(
```

Comment:
is this only for doc level monitor? if yes, plz rename method to reflect that

Reply:
They could be used for other monitors too

Reply:
the trigger you used is a doc level trigger

Reply:
Ah.. it might make sense to allow Alert with null trigger because of this usecase